README.md

```diff
@@ -7,6 +7,7 @@
 **C# Microservices for [Skyra]**
 
 [![GitHub](https://img.shields.io/github/license/skyra-project/arkadia)](https://github.com/skyra-project/arkadia/blob/main/LICENSE.md)
+[![codecov](https://codecov.io/gh/skyra-project/arkadia/branch/main/graph/badge.svg?token=DYhRX6ailX)](https://codecov.io/gh/skyra-project/arkadia)
```

Comment:
```suggestion
[![codecov](https://codecov.io/gh/skyra-project/arkadia/branch/main/graph/badge.svg?token=QEmhhyzHW5)](https://codecov.io/gh/skyra-project/arkadia)
```

Was the wrong token before. They are repo-specific and retrieved from the Codecov settings.